test/test_primitives.cpp

```diff
@@ -71,11 +71,10 @@ TORRENT_TEST(primitives)
 	// on failing announces
 	announce_entry ae("dummy");
 	int last = 0;
-	aux::session_settings sett;
-	sett.set_int(settings_pack::tracker_backoff, 250);
+	int const tracker_backoff = 250;
```

Comment:
if you wouldn't mind, this could be `auto const tracker_backoff = seconds(250);`